doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>Here a bit of explanation what the slc is for would be nice. maybe also link to slc repo or docu</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>